src/components/v5/common/ActionFormRow/types.ts

```diff
@@ -11,5 +11,9 @@ export interface ActionFormRowProps {
   children?:
     | ((props: UseToggleReturnType) => React.ReactNode)
     | React.ReactNode;
-  tooltip?: TooltipProps['tooltipContent'];
+  // tooltip?: TooltipProps['tooltipContent'];
```

Comment:
if this is commented on purpose, please write a comment

Reply:
Fixed